openmmapi/include/openmm/internal/vectorize_neon.h

```diff
@@ -101,6 +109,14 @@ class fvec4 {
     void store(float* v) const {
         vst1q_f32(v, val);
     }
+
+    /// Store only the lower three elements of the vector.
+    void storeVec3(float* v) const {
```

Comment:
This name is potentially confusing, since people will assume it refers to the `Vec3` class.  Could we call it something different?

Reply:
Happy for a better name to be used, but not sure what it should be. Suggestions welcome. It's storing/reducing to 3 elements in memory, so Vec3 seemed the best fit.